lib/mayaUsd/ufe/Utils.cpp

```diff
@@ -152,12 +152,8 @@ std::string uniqueName(const TfToken::HashSet& existingNames, std::string srcNam
 	return dstName;
 }
 
-std::string uniqueChildName(const Ufe::SceneItem::Ptr& parent, const Ufe::Path& childPath)
+std::string uniqueChildName(const UsdSceneItem::Ptr& usdParent, const Ufe::Path& childPath)
```

Comment:
We always have a UsdSceneItem in maya-usd, no sense in passing around a base class object and downcasting.